shapely/geometry/multilinestring.py

```diff
@@ -11,33 +11,31 @@
 
 class MultiLineString(BaseMultipartGeometry):
     """
-    A collection of one or more line strings
+    A collection of one or more LineStrings.
 
     A MultiLineString has non-zero length and zero area.
 
+    Parameters
+    ----------
+    lines : sequence
+        A sequence LineStrings, or a sequence of line-like coordinate
+        sequences or array-likes (see accepted input for LineString).
+
     Attributes
     ----------
     geoms : sequence
         A sequence of LineStrings
+
+    Examples
+    --------
+    Construct a collection containing two LineStrings.
```

Comment:
```suggestion
    Construct a MultiLineString containing two LineStrings.
```

(using concrete geometry types for consistency with the GeometryCollection example above)